pkgs/development/tools/analysis/codeql/default.nix

```
@@ -34,6 +34,7 @@ stdenv.mkDerivation rec {
 
     ln -sf $out/codeql/tools/linux64/lib64trace.so $out/codeql/tools/linux64/libtrace.so
 
+    sed -i 's%\$CODEQL_DIST/tools/\$CODEQL_PLATFORM/java-aarch64%\${jdk11}%g' $out/codeql/codeql
     sed -i 's%\$CODEQL_DIST/tools/\$CODEQL_PLATFORM/java%\${jdk11}%g' $out/codeql/codeql
```

You could probably avoid a `sed` call like this, but it's a minor nit and it'll be fine either way

```suggestion
    sed -i 's%\$CODEQL_DIST/tools/\$CODEQL_PLATFORM/java\(-aarch64\)\?%\${jdk11}%g' $out/codeql/codeql
```

---

I think I prefer the current solution with two seds as it is clear that first the -aarch64 and then the other ocurence needs to be replaced.